include/fsm/fsm.h

```diff
@@ -32,6 +32,13 @@ typedef unsigned int fsm_state_t;
  * their end states. These can be used to determine which of the
  * original FSM(s) matched when executing a combined FSM. */
 typedef unsigned int fsm_end_id_t;
+#define FSM_END_ID_NONE ((unsigned int)-1)
```

Comment:
Is this used? I don't see it elsewhere in this diff

Reply:
Good catch. I think when I switched from a "get the ID" to "get the set of IDs" interface this was no longer used, but I forgot to remove it.

Fixed in 7bfffb0f.